vpr/src/place/place.cpp

```diff
@@ -325,6 +326,7 @@ static e_move_result try_swap(float t,
                               t_placer_prev_inverse_costs* prev_inverse_costs,
                               float rlim,
                               MoveGenerator& move_generator,
+                              ManualMoveGenerator& manual_move_generator,
```

Comment:
Why do you need to have two different parameters: MoveGenerator and ManualMoveGenerator while MoveGenerator is the base class of ManualMoveGenerator?